go/genesis/api/api.go

```diff
@@ -80,6 +81,53 @@ func (d *Document) WriteFileJSON(filename string) error {
 	return nil
 }
 
+// SanityCheck does basic sanity checking on the contents of the genesis document.
+func (d *Document) SanityCheck() error {
+	if d.Height < 0 {
```

Comment:
Shouldn't we just make the height an `uint64`?

Reply:
I think it's set as `int64` for compatibility with Tendermint -- having it as `uint64` might just overflow silently.